cost-analyzer/templates/cost-analyzer-network-costs-template.yaml

```diff
@@ -36,7 +36,7 @@ spec:
 {{- end }}
 {{- if .Values.networkCosts.resources }}
         resources:
-          {{ toYaml .Values.networkCosts.resources | indent 10 }}
+{{ toYaml .Values.networkCosts.resources | indent 10 }}
```

Comment:
@michaelmdresser Wasn't this the same line you pointed out in the previous PR? Should've asked you to test it - I had no problems reproducing the issue today. 😞